Makefile.am

```
@@ -276,6 +276,7 @@ test_run_tests_SOURCES = test/blackhole-server.c \
                          test/test-tcp-writealot.c \
                          test/test-tcp-write-fail.c \
                          test/test-tcp-try-write.c \
+	                 test/test-tcp-write-in-a-row.c \
```

```suggestion
                         test/test-tcp-write-in-a-row.c \
```
(I think - indent error at any rate)